web/src/app/page.tsx

```diff
@@ -14,10 +14,26 @@ import {
   faMemory,
   faCheck,
 } from "@fortawesome/free-solid-svg-icons";
-import { useState } from "react";
+import { useEffect, useMemo, useState } from "react";
 
 export default function Home() {
   const router = useRouter();
+  const [copied, setCopied] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const [proverCommand, setProverCommand] = useState(
+    "export curl -sSL https://b4vue13qqtcrf2u0vi7ko0jdu0.ingress.akash-palmito.org/0k-worker.sh | sh",
```

Comment:
Is this URL not in an env variable? (https://b4vue13qqtcrf2u0vi7ko0jdu0.ingress.akash-palmito.org)